website/versioned_docs/version-5.x/ecosystem-snowpack.md

```diff
@@ -0,0 +1,102 @@
+
+## Configuration
+
+Modify the `index.js` file to not mount your app immediately, but rather to export the single-spa lifecycles. If using Vue, for example, see https://single-spa.js.org/docs/ecosystem-vue#usage.
+
+The following Snowpack config can be used as the basis for a single-spa + Snowpack setup. It requires installing [systemjs-webpack-interop](https://github.com/joeldenning/systemjs-webpack-interop) and `@snowpack/plugin-webpack`:
+
+```sh
+npm install --save-dev systemjs-webpack-interop @snowpack/plugin-webpack
+
+yarn add --dev systemjs-webpack-interop @snowpack/plugin-webpack
+
+pnpm install --save-dev systemjs-webpack-interop @snowpack/plugin-webpack
+```
+
+```js
+const { merge } = require("webpack-merge");
+const SystemJSPublicPathWebpackPlugin = require("systemjs-webpack-interop/SystemJSPublicPathWebpackPlugin");
+
+/** @type {import("snowpack").SnowpackUserConfig } */
+module.exports = {
+  mount: {
+    /* ... */
+  },
+  plugins: [
+    [
+      "@snowpack/plugin-webpack",
+      {
+        extendConfig(config) {
+          delete config.optimization.runtimeChunk;
+          delete config.optimization.splitChunks;
+
+          return merge(config, {
+            mode: "development",
+            module: {
+              rules: [
+                {
+                  parser: {
+                    system: false,
+                  },
+                },
+              ],
+            },
+            output: {
+              libraryTarget: "system",
+            },
+            plugins: [
+              new SystemJSPublicPathWebpackPlugin({
+                systemjsModuleName: "snowpack-test",
+                rootDirectoryLevel: 2,
+              }),
+            ],
+          });
+        },
+      },
+    ],
+  ],
+  routes: [
+    /* Enable an SPA Fallback in development: */
+    // {"match": "routes", "src": ".*", "dest": "/index.html"},
+  ],
+  optimize: {
+    /* Example: Bundle your final build: */
+    // "bundle": true,
+  },
+  packageOptions: {
+    /* ... */
+  },
+  devOptions: {},
+  buildOptions: {
+    baseUrl: "http://localhost:8080/",
+  },
+};
+```
+
+## Local development
+
+Snowpack works well with [development via import map overrides](https://single-spa.js.org/docs/recommended-setup#local-development). You should use http://localhost:8080/index.js as the URL for your import map override.
+
+Static Assets currently do not load from the correct URL in development, pending a PR to Snowpack: https://github.com/snowpackjs/snowpack/pull/2407. However, static assets do load from the correct URL in production, due to systemjs-webpack-interop.
```

Comment:
This could be made into an info or warning callout using this syntax https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions

Reply:
👍  updated